src/bindings/fluid_ladspa.c

```diff
@@ -951,46 +956,40 @@ int fluid_ladspa_check(fluid_ladspa_fx_t *fx, char *err, int err_size)
     int i;
```

Comment:
This is a comment on the line above "@ return FLUID_OK on success, otherwise FLUID_FAILED" which I cannot comment on:

It it not clear that FLUID_FAILED is returned when a "problem" is detected.

Reply:
Áck, fixed